docs/installation_guide.rst

```diff
@@ -0,0 +1,92 @@
+.. _installation_guide:
+.. _intro_installation_guide:
+
+******************
+Installation Guide
+******************
+
+Welcome to the |project| Installation Guide!
+
+
+Installing |project|
+====================
+
+This page describes how to install |project| on different platforms.
+
+.. contents::
+  :local:
+
+Prerequisites
+--------------
+You need Python 2.7 or 3.5+
+
+pylibssh requires libssh to be installed in particular:
+
+- libssh version 0.9.0 and later.
+
+  To install libssh refer to its `Downloads page
+  <https://www.libssh.org/get-it/>`__.
+
+.. _from_pip:
+
+Installing |project| with ``pip``
+---------------------------------
+
+|project| can be installed with ``pip``, the Python package manager.  If ``pip`` isn't already available on your system of Python, run the following commands to install it::
+
+    $ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+    $ python get-pip.py --user
+
+Then install |project|:
+
+.. parsed-literal::
+
+    $ pip install --user |project|
+
+.. note::
+
+    Running ``pip`` with ``sudo`` will make global changes to the system. Since ``pip`` does not coordinate with system package managers, it could make changes to your system that leaves it in an inconsistent or non-functioning state. This is particularly true for macOS. Installing with ``--user`` is recommended unless you understand fully the implications of modifying global files on the system.
+
+.. note::
+
+    Older versions of ``pip`` default to http://pypi.python.org/simple, which no longer works.
```

Comment:
This is not really true, there's a redirect.

TODO: Stop vendoring a copy of the PyPA guides, link there instead.